cocos/spine/skeleton-data.ts

```diff
@@ -207,20 +207,22 @@ export class SkeletonData extends Asset {
             }
             return null;
         }
-        const spData = spine.wasmUtil.querySpineSkeletonDataByUUID(this._uuid);
+
+        let uuid = this.mergedUUID();
```

Comment:
Why need to use new uuid?

Reply:
After replace resource, the atlas or json file has been changed.
But the uuid of the asset is not changed.